includes/Dashboard.php

```diff
@@ -165,12 +165,17 @@ public function load_stories_dashboard() {
 		 */
 		$preload_paths = apply_filters( 'web_stories_dashboard_preload_paths', $preload_paths );
 
+		$_GET['_embed'] = 1;
```

Comment:
Shouldn't this be 'author'?

Reply:
it feels like the check should just be for `_embed` being truthy so that we can get other embedded data in the future, but not get ALL of it back if we don't need it.